test/provider-vault-installer.yaml

```diff
@@ -0,0 +1,50 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: secrets-store-csi-driver-provider-vault
+---
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  labels:
+    app: csi-secrets-store-provider-vault
+  name: csi-secrets-store-provider-vault
+spec:
+  updateStrategy:
+    type: RollingUpdate
+  selector:
+    matchLabels:
+      app: csi-secrets-store-provider-vault
+  template:
+    metadata:
+      labels:
+        app: csi-secrets-store-provider-vault
+    spec:
+      serviceAccountName: secrets-store-csi-driver-provider-vault
+      containers:
+        - name: secrets-store-csi-driver-provider-vault
+          image: e2e/secrets-store-csi-driver-provider-vault:latest
+          imagePullPolicy: Never
+          args:
+            - --endpoint=/provider/vault.sock
```

Comment:
I see above the socket is specified at `/tmp/vault.sock` but here it's `/provider`. Is this correct?

Reply:
Intentional - discussion in comment below.